packages/create-cli/src/utils/directory.ts

```diff
@@ -33,3 +33,13 @@ export function usePackageName (packageName: string) {
 export function hasGitIgnore () {
   return fs.existsSync('./.gitignore')
 }
+
+export function isValidUrl (urlString: string) {
```

Comment:
Could this use [the URL constructor](https://developer.mozilla.org/en-US/docs/Web/API/URL) instead of a long regex? @nahuelon